react/ProductSummaryReference.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```

Comment:
@HugoLaguna can you lint this file?

Reply:
with `vtex.product-summary-context@0.9.0` already hasn't warnings, was missing types. can you check ?
added CHANGELOG!

Reply:
wait, are talking about this ? i don't know how fix it, i'm locking for a solution
![image](https://user-images.githubusercontent.com/33559104/132814780-12a6880a-c37a-4a09-9cda-a019342ef646.png)

Reply:
i fixed this alerts 
![image](https://user-images.githubusercontent.com/33559104/132817300-d53fa569-76db-45d8-85f0-b3f09cc64182.png)